include/wx/cmdargs.h

```diff
@@ -31,19 +31,17 @@ class WXDLLIMPEXP_BASE wxCmdLineArgsArray
     wxCmdLineArgsArray() { m_argsA = NULL; m_argsW = NULL; }
 
     template <typename T>
-    wxCmdLineArgsArray& operator=(T **argv)
+    void Init(int argc, T **argv)
     {
         FreeArgs();
 
         m_args.clear();
+        m_args.reserve(argc);
 
-        if ( argv )
+        for ( int i = 0; i < argc; i++ )
         {
-            while ( *argv )
-                m_args.push_back(*argv++);
+            m_args.push_back(*argv++);
```

Comment:
To be honest, I don't think there is any real difference between the two forms, but I'll update it to use the index.